src/tinymce.js

```diff
@@ -54,29 +54,46 @@ angular.module('ui.tinymce', [])
         expression = {};
 
         angular.extend(expression, scope.$eval(attrs.uiTinymce));
+        tinymceWatchers.push(scope.$on('tinymce:init', function (ev, editorInstance) {
+          ev.stopPropagation();
+          ngModel.$render();
+          ngModel.$setPristine();
+          if (form) {
+            form.$setPristine();
+          }
+        }));
+        tinymceWatchers.push(scope.$on('tinymce:ExecCommand', function (ev, editorInstance) {
```

Comment:
If this extra layer of eventing is needed, I suggest breaking the commons ones out into a helper function.

If we need these and we take this patch I can make that change in the dev branch.